### PR TITLE
Update for atomic registry use case

### PR DESCRIPTION
**Pull request by aweiteka**

### CODE REVIEW

**Comment by aweiteka** [comment]
@adellape I removed WIP flag. Please review.

File: admin_guide/manage_authorization_policy.adoc [hunk 1]
```diff
@@ -16,6 +16,12 @@ policies] and the administrator CLI to manage the
 link:../architecture/additional_concepts/authorization.html[roles and bindings]
 within a policy.
 
+ifdef::atomic-registry[]
+The web console also provides some basic management of authorization roles.
+Through the web console users and groups may be assigned **registry-admin**,
+**regisry-editor** or **registry-viewer**.
```
Comment (adellape) [inline admin_guide/manage_authorization_policy.adoc]: s/regisry/registry/

File: admin_guide/service_accounts.adoc [hunk 2]
```diff
@@ -11,6 +11,7 @@ toc::[]
 
 == Overview
 
+ifdef::openshift-origin,openshift-origin,openshift-enterprise,openshift-dedicated[]
```
Comment (adellape) [inline admin_guide/service_accounts.adoc]: Got a double `openshift-origin` in here.

File: architecture/additional_concepts/authorization.adoc [hunk 3]
```diff
@@ -160,6 +160,7 @@ a set of default roles that can be added to users and groups in the
 link:#cluster-policy-and-local-policy[cluster policy] or in a
 link:#cluster-policy-and-local-policy[local policy].
 
+ifdef::openshift-origin,openshift-origin,openshift-enterprise,openshift-dedicated[]
```
Comment (adellape) [inline architecture/additional_concepts/authorization.adoc]: Double `openshift-origin`

File: architecture/additional_concepts/authorization.adoc [hunk 4]
```diff
@@ -238,6 +273,7 @@ using:
 $ oadm policy reconcile-cluster-roles
 ----
 endif::[]
+ifdef::openshift-origin,openshift-origin,openshift-enterprise,openshift-dedicated[]
```
Comment (adellape) [inline architecture/additional_concepts/authorization.adoc]: Looks like the double `openshift-origin` is throughout. Not sure if one was meant to be `openshift-online`. If so, consider that the `openshift-online` ifdef is not needed for any `install_config` or `admin_guide` topics.

File: architecture/additional_concepts/authorization.adoc [hunk 5]
```diff
@@ -300,7 +300,7 @@ endif::[]
 ifdef::openshift-dedicated[]
 . Running of privileged containers.
 endif::[]
-ifdef::openshift-origin,openshift-origin,openshift-enterprise,openshift-dedicated[]
+ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
```
Comment (adellape) [inline architecture/additional_concepts/authorization.adoc]: The 3 new ifdefs in this Arch topic can also include `openshift-online`, I think.

**Comment by adellape** [comment]
@aweiteka Oops I commented directly on one of the commits on accident: https://github.com/openshift/openshift-docs/pull/1848/files/de09df7f2f574ed8bb3bc9197861b4904cb1b702..4f78e2766d2dd0cb5d18864e1f0d57e88310731e#r60470622

Other than that, did some spot checking in a local build and it LGTM.

**Comment by adellape** [comment]
@aweiteka After those remaining ifdefs are update, plz squash. :bow:

**Comment by aweiteka** [comment]
@adellape squashed. All set.

**Comment by adellape** [comment]
no revhistory needed

**Pull request merged**